docs/DeveloperGuide.md

```diff
@@ -504,3 +466,54 @@ testers are expected to do more *exploratory* testing.
 2. Dealing with corrupted data files 
    1. Add a new field `"newField" : "newField"` to a client. Expected: All data is lost. The app starts on a clean slate.
    2. Remove `"insurancePlans"` field from a client. Expected: All data is lost. The app starts on a clean slate.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix : Planned Enhancements**
+Do note we have 4 members in our group.
+
+### 1. addClaim ID
+A single claim ID can be added to multiple users. i.e. `B1100` can be added to both client `A` and client `B` with no error.
+This will be fixed in a future version such that claim ID is unique across all clients.
+
+### 2. addClaim amount
+Claim amount can currently exceed 1 million. In the future, a restriction will be placed on the claim amount such that if
+the claim amount is over 1 million, it will be rejected with an appropriate error message. This is in line with our
+restriction on claim amount (claim amount cannot exceed 1 million)
+
+### 3. Add command and Edit Command does not allow clients to have same names yet allows two people to have the same contact details.
```

Comment:
Inconsistent full stops for all the points

Reply:
good catch